styles/app.css

```diff
@@ -0,0 +1,139 @@
+body {
+  margin: 0;
+  font-family: sans-serif;
+}
+
+ul {
+  padding-left: 0;
+  list-style: none;
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.navbar {
+  background: #2CC5E1;
+  overflow: auto;
+  margin: auto;
+}
+
+.navbar-left {
+ float: left;
+ display: inline-block;
+ color: white;
+padding-bottom: 1rem;
+}
+
+.navbar-right {
+ float: right;
+ display: inline-block;
+}
+
+.navbar-right__item {
+  float: right;
+  display: inline-block;
+  padding-top: 2rem;
```

Comment:
Instead of using padding-top to line things up vertically, when we use float (or inline blocks), we can use `line-height` to help vertically center things. Try setting `line-height: 2rem `on both this selector and the selector for `.navbar-left`